models/social_connection.py

```diff
@@ -0,0 +1,69 @@
+import json
```

Comment:
To add to your 1:1 argument, Robots are 1:1 too (which we have separate Models for), but they grow linearly with the number of years. Connections don't.